src/pages/ModuleEditor/ModuleEditor.tsx

```diff
@@ -133,6 +160,16 @@ export default function ModuleEditor({match: {params: {moduleId}}}: Props) {
               <Form.Label column={true}>Module Description</Form.Label>
               <Input name={propertyOf<ModuleForm>('description')} placeholder='Description' type='textarea' disabled={!editing}/>
             </Form.Group>
+            <Form.Label column={true}>Tags</Form.Label>
+            <TagEditor
+              tags={moduleTags.map(mt => mt.tag)}
+              tagSuggestions={tagSuggestions}
+              mes={message?.variant}
+              editing={editing}
+              onAdd={onAddTag}
+              onDelete={onDeleteTag}
+              onInput={onTagInput}
+            />
```

Comment:
You should use formik's values instead of rolling your own.
```suggestion
            <FieldArray name={propertyOf<ModuleForm>('moduleTags')}>
              {(helpers) => (
                <TagEditor
                  tags={values.moduleTags.map(mt => mt.tag)}
                  tagSuggestions={tagSuggestions}
                  mes={message?.variant}
                  editing={editing}
                  onAdd={t => helpers.push(cast<ModuleTag>({moduleId: Number(moduleId), tagId: (t.id === 0) ? t.id : 0, tag: t}))}
                  onDelete={i => helpers.remove(i)}
                  onInput={onTagInput}
                />
              )}
            </FieldArray>
```